hbase-client/src/main/java/org/apache/hadoop/hbase/exceptions/MasterStoppedException.java

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
The header has been incorrectly formatted. Please do a copy-paste from other file to fix it?